integration-cli/docker_api_containers_test.go

```diff
@@ -442,7 +442,8 @@ func (s *DockerSuite) TestContainerAPITop(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	defer cli.Close()
 
-	top, err := cli.ContainerTop(context.Background(), id, []string{"aux"})
+	// sort by comm[andline] to make sure order stays the same in case of PID rollover
+	top, err := cli.ContainerTop(context.Background(), id, []string{"aux", "--sort", "comm"})
```

Comment:
Only problem here is not all `ps` implementations support `--sort`.
Maybe we can check for either `sh` or `top` in the first assertion, and then the other in the next assertion.

Reply:
1. Note that the ps is run on the host, not inside container.
2. If you refer to ps from busybox, it won't work either since it ignores the `aux` flags and so the expected fields won't be there.
3. I looked around and can't find a version of `ps` which correctly supports `aux` but does not support `--sort=comm`. The obvious exception is busybox (e.g. alpine) but see 2.

Reply:
I took a look, flag `--sort` in both procps and procps-ng can be traced back to the first CVS commit in Feb 2002, and it seems it supported `--sort=comm` at that time. For the pre-CVS time, I checked sources of procps-0.99 (from April 1996) and the --sort flag is there, but it looks like long sort field specs are not supported.

Reply:
if order is not important, we could also just split the output and check if not `top` and `/bin/sh -c top` occur in the output (at any position)

Reply:
I think this PR is good as it is, unless there is a version of `ps` which supports `aux` but does not support `--sort`. I was not able to find such a beast.

Reply:
@cpuguy83 PTAL